src/core/qgsjsonutils.cpp

```diff
@@ -401,8 +401,8 @@ json QgsJsonUtils::jsonFromVariant( const QVariant &val )
   }
   else if ( val.type() == QVariant::Type::List )
   {
-    const auto vList{ val.toList() };
-    auto jList { json::array() };
+    const auto vList = val.toList();
+    auto jList = json::array();
```

Comment:
This could be made in `qgsspatialiteprovider.cpp::4410` and int `testqgsjsonuutils` as well, no?

Reply:
@signedav is [that](https://github.com/qgis/QGIS/pull/30758/commits/5238f84e7bf950d7d2401d971f9cafe57f3a75c4) the change you were thinking

Reply:
Yes. Thanks.